configuration/configuration_blackbox_test.go

```diff
@@ -141,3 +150,49 @@ func TestGetKeycloakEndpointUserInfoSetByEnvVaribaleOK(t *testing.T) {
 	assert.Nil(t, err)
 	assert.Equal(t, "userinfoEndpoint", url)
 }
+
+func TestGetTokenPrivateKeyFromConfigFile(t *testing.T) {
+
+	envKey := generateEnvKey(varTokenPrivateKey)
+	realEnvValue := os.Getenv(envKey) // could be "" as well.
+
+	os.Unsetenv(envKey)
+	defer func() {
+		os.Setenv(envKey, realEnvValue)
+		resetConfiguration()
```

Comment:
Why not pass the `*testing.T` to the `resetConfiguration` function and have the test fail rather than panic?